external-crates/move/crates/move-analyzer/editors/code/scripts/create.sh

```diff
@@ -55,7 +54,7 @@ SUPPORTED_OS[macos-arm64]=darwin-arm64
 SUPPORTED_OS[linux-x86_64]=linux-x64
 #SUPPORTED_OS[windows-x86_64]=win32-x64
 
-TMP_DIR=$( mktemp -d -t vscode-create )
+TMP_DIR=$( mktemp -d -t vscode-createXXX )
```

Comment:
What's that for?

Reply:
Without this it fails on linux with `mktemp: too few X's in template "vscode-create"` it will replace the X's with random chars